src/cmd/team_test.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/opslevel/opslevel-go/v2024"
+)
+
+const (
+	teamFileName = "test_team.yaml"
+	teamName     = "CLI Test Team"
+)
+
+func Test_TeamCRUD(t *testing.T) {
```

Comment:
This would be much more readable if divided into functions like `Test_TeamCreate`, `Test_TeamUpdate` and so on, I would be able to quickly see what team related functions have tests by glancing at the functions defined in this file.

Reply:
The code could be divided up into smaller functions but we can't run something like `Test_TeamUpdate` in isolation. We first need to create some test team to update